scripts/tests/test_suites.sh

```diff
@@ -249,9 +249,9 @@ for j in "${iter_array[@]}"; do
             cat <(timeout 1 dns-sd -B _matterc._udp)
         fi
         echo "          * Pairing to device"
-        ${tool_run_prefix} "${test_case_wrapper[@]}" out/debug/standalone/chip-tool pairing qrcode "$node_id" MT:D8XA0CQM00KA0648G00 | tee "$pairing_log_file"
+        ${tool_run_prefix} "${test_case_wrapper[@]}" out/debug/standalone/chip-tool pairing qrcode "$node_id" MT:D8XA0CQM00KA0648G00 --commissioner-name beta | tee "$pairing_log_file"
```

Comment:
can we just use the default? Wondering if this is really needed as I am actively changing the test_suites.sh to  pyhon and am unsure what changes wille be relevant and what not

Reply:
The specific reason why I have added beta here is because while working on the next set of patches for having a YAML test with multiple admin I found out that `beta` was the one that was not working as of today when you have multiple controllers instances in the same process.


The first issue is that the delegates for both dns-sd and the exchange manager are beeing overriden by the last instance (see #12261 for dns-sd and I have a local patch coming for the exchange manager case).

So the only instance that was able to pair was `gamma`. The last one that is created.

But then, `gamma` was using the wrong fabric index, so beside pairing it was not working for sending commands (see #12257).

So, before fixing anything, `gamma` was the only one to work - but it was only half working. `alpha` and `beta` were not able to pair.
After fixing the delegates, `alpha` was working properly, `beta` and `gamma` were half working.
After fixing all those, all of them works properly. I figured out that `beta`, since it is not the first and not the last commissioner, does not benefits from side effects of the code and decide that it is a good candidate for using as default.


The last commissioner (`gamma`) is currently working on top of tree because it currently override the delegates used for both dns-sd and the exchange manager. But is it sort of broken as it uses the fabric index of `alpha` (see #12264).
 
Once the delegates was fixed (see #12261 and I have a local patch that I need to clean up for the exchange manager delegate bits), `alpha`

Reply:
Seems like your changes has landed... I have removed the commissioner-name stuff here. I will add something back when adding directly the YAML test I guess.